component/background/RoomService.tsx

```diff
@@ -76,29 +78,29 @@ class RoomService {
     if (!data) throw new Error(`data is {data}`);
     this.messageRef.push().set({
       data,
-      fromID: this.myConfRef.key
+      fromID: this.id
     });
   }
 
-  public getUser(userID: string): Promise<any> {
-    return this.getDataAtReference(this.userListRef.child(userID));
+  public getUser(userID: string): Promise<void> {
```

Comment:
Since this is a public method that could be called upon by other developers, I think there should be jsdocs here.